onnxruntime/core/platform/env.cc

```diff
@@ -19,6 +19,20 @@ limitations under the License.
 
 namespace onnxruntime {
 
+std::ostream& operator<<(std::ostream& os, const ThreadOptions::ThreadAffinity& aff) {
+  os << "{";
+  std::copy(aff.logical_proc_ids.cbegin(), aff.logical_proc_ids.cend(), std::ostream_iterator<int>(os, " "));
+  return os << "}";
+}
+
+std::ostream& operator<<(std::ostream& os, gsl::span<const ThreadOptions::ThreadAffinity> affs) {
+  os << "{";
+  for (const auto& aff : affs) {
+    os << aff;
```

Comment:
> os << aff; [](http://example.com/codeflow?start=3&length=11)

no split in any form?

Reply:
I understand each "aff" will be further embraced by "{..}", but how about at lease a comma in between?